lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -582,6 +582,12 @@ module.exports = class MethodMappings {
 
           return value;
         },
+        async isElementPresent(webElementOrId) {
+          const element = await this.getWebElement(webElementOrId);
+          if (!element) return false;
+          return element instanceof WebElement || element instanceof ShadowRoot;
+        },
```

Comment:
`getWebElement` never returns an instance of ShadowRoot as of now

Reply:
Thanks, I've Updated it, But the PR Doesn't get Updated Automatically, You can Check the forked Repo and see if it is fixed